src/main/steps/urls.ts

```diff
@@ -96,6 +96,7 @@ export const YOUR_COMMENTS_SENT: PageLink = '/your-comments-sent';
 export const NO_RESPONSE_YET: PageLink = '/no-response-yet';
 
 export const APPLICATION_SUBMITTED: PageLink = '/application-submitted';
```

Comment:
maybe this should be a function instead that takes in the applicationType and returns the correct url accordingly